### ISSUE TITLE
CLI to load?

### ISSUE THREAD

**Issue by bgehman** [nolanlawson/pouchdb-dump-cli#20]
Nice to have a CLI to dump (ie: backup) C/PouchDB databases, but is there a CLI to load (ie: restore) a database dump produced by pouchdb-dump-cli ?

**Comment by fiatjaf** [comment]
Yes. It is very clearly said in the README: https://github.com/pouchdb-community/pouchdb-load
